cmd/exo/cmd/privnet_create.go

```diff
@@ -102,15 +116,23 @@ func privnetCreate(name, desc, zoneName string) error {
 	newNet := resp.(*egoscale.Network)
 
 	table := table.NewTable(os.Stdout)
-	table.SetHeader([]string{"Name", "Description", "ID"})
-	table.Append([]string{newNet.Name, newNet.DisplayText, newNet.ID.String()})
+	table.SetHeader([]string{"Name", "Description", "ID", "DHCP"})
+
+	table.Append([]string{
+		newNet.Name,
+		newNet.DisplayText,
+		newNet.ID.String(),
+		dhcpRange(startIP, endIP, netmask),
+	})
 	table.Render()
 	return nil
 }
 
 func init() {
-	privnetCreateCmd.Flags().StringP("name", "n", "", "Private network name")
 	privnetCreateCmd.Flags().StringP("description", "d", "", "Private network description")
+	privnetCreateCmd.Flags().StringP("startip", "s", "", "the beginning IP address in the network IP range. Required for managed networks.")
+	privnetCreateCmd.Flags().StringP("endip", "e", "", "the ending IP address in the network IP range. Required for managed networks.")
+	privnetCreateCmd.Flags().StringP("mask", "m", "", "the netmask of the network.  Required for managed networks")
```

Comment:
"mask", could be `UintP`?

Reply:
yes, updated ;) thanks

Reply:
I'd still accept both though.

Reply:
You want `mask` and `netmask` ?

Reply:
Now, the command supports both.

Reply:
@mcorbin the minimum is to be as good as the API, then we can do more but I wouldn't do things differently.